README.rst

```diff
@@ -48,6 +48,9 @@ the folowing to enable autocompletion for hass-cli commands.
 
   $  source <(hass-cli completion zsh)
 
+A Note about Python Virtual Environments
+========================================
+In the 0.4 release we made a mistake on the pyyml release so there would be a mismatch in dependencies in relation to Home Assistant but past and future releases should in theory be fine. That said - if you want the least amount of surprises then a separate venv or using pipsi like : `pipsi install --python python3 homeassistant-cli` is best.
```

Comment:
It's `pyyaml` not `pyyml`. 

We should simply add details about the venv. I don't see any value to add information about an old release.

Reply:
also with 0.5 I just released this issue is gone.